### DISCUSSION TITLE
_thisS.options.overloadTranslationOptionHandler is no function

### DISCUSSION THREAD

**Discussion by GabrielDRKZ** [i18next/i18next#1926] [Q&A]
https://media.discordapp.net/attachments/1086055127233871982/1086055535788445739/image.png?width=332&height=43 - está dando erro quando eu tento executar um comando - https://media.discordapp.net/attachments/1086055127233871982/1086055563705720932/image.png?width=249&height=238

**Comment by jamuhl** [comment]
Did you init i18next before calling t function?

**Reply by GabrielDRKZ** [reply]
Yes

**Reply by GabrielDRKZ** [reply]
https://prnt.sc/Gq575st2lp4Y

**Reply by jamuhl** [reply]
are you sure all the async stuff runs in correct timings?

are you using this on server? you might have a look at...https://github.com/i18next/i18next-http-middleware (highly recommended when using node as web server - as chances are high you will run into issues with your solution on concurrent requests)

**Reply by GabrielDRKZ** [reply]
Mudei de lugar e parou de dar erro, realmente não estava iniciando como eu achei que estava
